README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/github/workflow/status/omnivector-solutions/armasec/test_on_push/main?label=main-build&logo=github&style=plastic
+   :alt: main build
+.. image:: https://img.shields.io/github/issues/omnivector-solutions/py-buzz?label=issues&logo=github&style=plastic
+   :alt: github issues
+.. image:: https://img.shields.io/github/issues-pr/omnivector-solutions/armasec?label=pull-requests&logo=github&style=plastic
+   :alt: pull requests
+.. image:: https://img.shields.io/github/contributors/omnivector-solutions/armasec?logo=github&style=plastic
+   :alt: github contributors
+
+.. image:: https://img.shields.io/pypi/pyversions/armasec?label=python-versions&logo=python&style=plastic
+   :alt: python versions
+.. image:: https://img.shields.io/pypi/v/armasec?label=pypi-version&logo=python&style=plastic
+   :alt: pypi version
+
+.. image:: https://img.shields.io/pypi/l/armasec?style=plastic
+   :alt: license
+
+.. raw:: html
+
+    <br />
+    <p align="center">
+      <a href="https://github.com/omnivector-solutions/armasec">
+        <img src="_static/logo.png" alt="Logo" width="80" height="80">
+        <h3 align="center">Armasec</h3>
+      </a>
+      <p align="center"><i>An Omnivector Solutions initiative</i><p>
+    </p>
```

Comment:
What about:
```suggestion
.. figure: _static/logo.png
   :alt: Logo
   :align: center
   :width: 80px
   
   An Omnivector Solutions initiave
```

Reply:
Let me try that. I just yanked that template html out of one of the other projects.

Reply:
Fixed in next commit